src/ci/docker/host-x86_64/dist-powerpc64-linux/shared.sh

```diff
@@ -8,7 +9,7 @@ exit 1
   trap "$on_err" ERR
   bash -c "while true; do sleep 30; echo \$(date) - building ...; done" &
```

Comment:
Looks like posix shell to me, we could probably just call `sh` here rather than `bash`. But I wonder if we even need to run that on another shell? Maybe we could?
```bash
running=1
while running
do
  sleep 30
  echo $(date) - building ...
done &
```
One thing about this is it is running with `set +x` which we may not want.

Reply:
Please do not make unrelated changes here. All these scripts have no tests and the failure scenario is that the silently stop running tests.

Reply:
I leave them as is.